core/src/test/java/com/graphhopper/GraphHopperTest.java

```diff
@@ -2504,16 +2500,6 @@ public void testBarriers() {
                 setMinNetworkSize(0);
         hopper.importOrLoad();
 
-        {
-            // the bollard blocks the road for bikes, and we need to take a big detour. note that this bollard connects
-            // two ways
-            GHResponse bikeRsp = hopper.route(new GHRequest(51.257709, 12.309269, 51.257594, 12.308882).setProfile("bike"));
-            assertEquals(1185, bikeRsp.getBest().getDistance(), 1);
-            // pedestrians can just pass the bollard
-            GHResponse footRsp = hopper.route(new GHRequest(51.257709, 12.309269, 51.257594, 12.308882).setProfile("foot"));
-            assertEquals(28, footRsp.getBest().getDistance(), 1);
-        }
-
```

Comment:
I think this test was testing some general barrier-node logic, not really related to any specific vehicles. Did it fail for this PR, and do you know why? I could try to find another example for the same case if this is not a barrier anymore or something.